tests/core/v5_1/test_v51_rpc_handlers.py

```diff
@@ -655,3 +652,106 @@ async def test_v51_rpc_sendFoundNodes_web3(
 
         assert second_receive.message.total == second_response.value
         assert encode_hex(second_receive.message.request_id) == request_id
+
+
+@pytest.mark.trio
+async def test_v51_rpc_recursiveFindNodes(tester, bob, make_request):
```

Comment:
@pipermerriam These tests are acting a little bit flaky - is there anything in this test and the one below that jumps out to you as problematic?